homework/generate-sequence/generateSequence.hpp

```diff
@@ -1,7 +1,16 @@
-#pragma once
+# pragma once
+
+#include <algorithm>
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> sequence;
```

Comment:
Reserve memory for vector, when you know how many elements will be inserted. 
This will avoid the situation with resizing vector few times (which is always expensive)

Reply:
Right, done.